R/segments.R

```diff
@@ -23,13 +23,18 @@
 #'   as_network()
 #' river <- bucharest_osm$river_centerline |> sf::st_geometry()
 #' delineate_segments(corridor, network, river)
+#' @srrstats {G2.16} This function checks numeric arguments for undefined values
+#'   (NaN, Inf, -Inf) and errors when encountering such values.
 delineate_segments <- function(corridor, network, river,
                                angle_threshold = 100) {
   # Check input
   checkmate::assert_class(corridor, "sfc_POLYGON")
   checkmate::assert_class(network, "sfnetwork")
   checkmate::assert_true(inherits(river, c("sf", "sfc")))
-  checkmate::assert_numeric(angle_threshold, lower = 90, upper = 180)
+  checkmate::assert_numeric(angle_threshold,
+                            lower = 90,
```

Comment:
I think this formally should be in the [0;180) range.
```suggestion
                            lower = 0,
```